spark-knn-examples/src/main/scala/com/github/saurfang/spark/ml/knn/examples/MNIST.scala

```diff
@@ -2,14 +2,16 @@ package com.github.saurfang.spark.ml.knn.examples
 
 import org.apache.spark.ml.Pipeline
 import org.apache.spark.ml.classification.KNNClassifier
-import org.apache.spark.ml.feature.{PCA, VectorAssembler}
-import org.apache.spark.ml.knn.KNN
+import org.apache.spark.ml.feature.PCA
 import org.apache.spark.mllib.util.MLUtils
-import org.apache.spark.sql.types._
 import org.apache.spark.sql.{DataFrame, Row, SQLContext}
-import org.apache.spark.{Logging, SparkConf, SparkContext}
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.log4j
+
+object MNIST {
+
+  val logger = log4j.Logger.getLogger(getClass)
 
-object MNIST extends Logging {
   def main(args: Array[String]) {
     val conf = new SparkConf()
```

Comment:
This should rather be : 
`val spark = SparkSession.builder().config("spark.serializer","org.apache.spark.serializer.KryoSerializer").getOrCreate()`
spark would be the SparkSession, and to access the SparkContext you would call `val conf = spark.sc`

BTW, the `config("spark.serializer","org.apache.spark.serializer.KryoSerializer")` isn't necessary anymore in Spark 2.0.0